### ISSUE TITLE
How to tell people how to compile?

### ISSUE THREAD

**Issue by ubsan** [leios/algorithm-archive#13]
Notably, for languages like C++, there are a few flags we might want to pass. Discussion here on how we'd like to format this.

**Comment by julianschacherpp** [comment]
Not quite the topic, but maybe general pages like "How to compile?" wouldn't be that bad.

**Comment by ubsan** [comment]
Yeah. A set of common flags would be good as well.

**Comment by leios** [comment]
Yeah, I think I will add a section on this (along with another page about makefiles) soon. Hopefully this week. We'll see. There's a lot to do! Thanks for the discussion.

**Comment by julianschacherpp** [comment]
I can provide an explanation of how to setup VisualStudio to get my C#-files working as well as an explanation of how to compile with the Linux/Windows-terminal.

**Comment by leios** [comment]
Do you mind if we come back to this in a bit when I start going through the different languages? I feel like that would be the best time to talk about language-specific compilation instructions.

I cannot really test anything on Windows, so I think this would be really helpful.

**Comment by ubsan** [comment]
seems fine to me.

**Comment by julianschacherpp** [comment]
No problem, I just wanted to have that said, so I can help you when you're working on this :)

**Comment by leios** [comment]
You guys rock! Thanks for all the help / Sorry I'm moving so slow.
